.travis.yml

```diff
@@ -0,0 +1,32 @@
+branches:
+  only:
+  - trunk
+
+# Default settings for build stages
+language: java
+jdk:
+  - openjdk8
+  - oraclejdk8
+  - oraclejdk9
+
+install: true
+script: ./gradlew build minJosmVersionClasses
+
+jobs:
+  include:
+    - stage: i18n
+      language: python
+      python: "3.5"
```

Comment:
The newest version of the transifex client is now ready for python 3.6.

And also with the newest version also this should be possible:
```
if [ ! -z "$TRANSIFEX_TOKEN" ]; then
  ./gradlew generatePot
  TX_TOKEN="$TRANSIFEX_TOKEN" tx push -s --no-interactive
fi
```
That's how I now do it with the wikipedia plugin.

Reply:
OK. I will change it to 3.6.